scripts/release/generate_release_maintenance_jobs.py

```diff
@@ -115,9 +121,12 @@ def get_trigger_missed_jobs_job_config(args, config, build_file):
         additional_data=data)
 
 
-def get_import_upstream_job_config(args, config, build_file):
-    template_name = 'release/deb/import_upstream_job.xml.em'
-    return _get_job_config(args, config, config.notify_emails, template_name)
+def get_import_upstream_job_config(args, config, build_file, package_format):
```

Comment:
Should the new argument be added with a default value so that it doesn't break external code potentially using it? (Not sure if we have done the same in other RPM related PRs.)

Reply:
I have not done that with other instances of this pattern.

I think it's an OK think to start doing, but If I understand correctly, an external python module couldn't import a function from this script anyway, since it isn't part of a module. Does that sound right?

Reply:
I don't mind either way. I could just see that maybe someone uses the API as is which would break. (But it could also just be nobody.)

Reply:
Right, in this specific case it is extremely unlikely that anyone is importing this file / API from it.

Reply:
Sounds like they'd be doing some unsupported stuff if they did manage to do the import, and since this would break the existing precedent, I'm inclined to leave it as-is.

Thanks for bringing it up.